enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tasks for subsidy requests app.
+"""
+
+import logging
+
+from celery import shared_task
+from django.apps import apps
+
+from enterprise_access.apps.api_client.discovery_client import DiscoveryApiClient
+from enterprise_access.apps.core.tasks import LoggedTaskWithRetry
+from enterprise_access.apps.subsidy_request.constants import SubsidyTypeChoices
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(base=LoggedTaskWithRetry)
+def update_course_title_for_subsidy_request_task(subsidy_type, subsidy_request_uuid):
+    """
+    Get course_title from lms and update subsidy_request with it
+    """
+    if subsidy_type == SubsidyTypeChoices.LICENSE:
```

Comment:
seems like we have a helper function `get_subsidy_model` that could be used here?

Reply:
this has been updated now